### PR TITLE
Updated termux requirements

### PR DESCRIPTION
**Pull request by roberts01**
Also maybe move this to the root of the Termux folder

### CODE REVIEW

File: helpers/termux_dependencies.py [hunk 1]
```diff
@@ -4,19 +4,21 @@
 def InstallTermuxDependencies() : 
     CustomPrint("Installing dependencies...")
     CustomPrint("Updating Termux")
-    os.system('apt update')
-    CustomPrint("Allow storage permission for storing extraced whatsapp.ab in interal storage.")
+    os.system('pkg update')
+    ustomPrint("Allow storage permission for storing extracted whatsapp.ab in interal storage:")
```
Comment (YuvrajRaghuvanshiS) [inline helpers/termux_dependencies.py:8]: CustomPrint()

**Comment by YuvrajRaghuvanshiS** [comment]
> 
> 
> Also maybe move this to the root of the Termux folder
No, that'd break the code.

File: helpers/termux_dependencies.py (hunk 1 above)
Comment (roberts01) [reply helpers/termux_dependencies.py:8]: oops my bad

**Comment by roberts01** [comment]
oh really? then dont it.
What would it break? I thought the Readme was the only file referencing it.

**Comment by YuvrajRaghuvanshiS** [comment]
> What would it break? I thought the Readme was the only file referencing it.

I have not worked on it for a while but I used to reference it to check and install dependencies if not installed automatically.

**Comment by YuvrajRaghuvanshiS** [comment]
I am getting confused with so many pulls, hard for me to keep track.

File: helpers/termux_dependencies.py [hunk 2]
```diff
@@ -4,19 +4,21 @@
...
-    os.system('apt update')
-    CustomPrint("Allow storage permission for storing extraced whatsapp.ab in interal storage.")
+    os.system('pkg update')
+    CustomPrint("Allow storage permission for storing extracted whatsapp.ab in interal storage:")
     os.system('termux-setup-storage')
-    CustomPrint("Done. Installing various required components.")
-    os.system('apt install curl grep tar proot wget -y')
-    CustomPrint("Done. Installing ADB for Termux")
+    CustomPrint("Done. Installing required dependencies...")
+    os.system('pkg install curl grep tar proot wget termcolor -y')
```
Comment (YuvrajRaghuvanshiS) [inline helpers/termux_dependencies.py:11]: termcolor is python module and can not be installed with pkg install. Don't merge python and shell requirements together.

File: helpers/termux_dependencies.py [hunk 3]
```diff
@@ -4,19 +4,21 @@
...
+        os.system('rm -r -f installjava') # Deleting any previous instance of installjava.
     except Exception as e : 
         pass
-    os.system('wget https://raw.githubusercontent.com/Hax4us/java/master/installjava && sh installjava')
-    CustomPrint("Done installing all dependencies. Now running proot.")
+    os.system('wget https://raw.githubusercontent.com/MasterDevX/java/master/installjava && sh installjava')
+    os.system('proot login')
+    CustomPrint("Connecting ADB with local device:")
+    os.system('adb connect localhost')
+    os.system('cd WhatsApp-Key-Databse-Extractor/')
```
Comment (YuvrajRaghuvanshiS) [inline helpers/termux_dependencies.py:21]: This file will not be run every time so there is no use of putting this here.

File: helpers/termux_dependencies.py (hunk 2 above)
Comment (YuvrajRaghuvanshiS) [inline helpers/termux_dependencies.py:11]: termcolor is python module and can not be installed with pkg install. Don't merge python and shell requirements together.

**Review by YuvrajRaghuvanshiS — CHANGES_REQUESTED** [review]

File: helpers/termux_dependencies.py (hunk 2 above)
Comment (roberts01) [reply helpers/termux_dependencies.py:11]: okay will sepearate that

File: helpers/termux_dependencies.py (hunk 3 above)
Comment (roberts01) [reply helpers/termux_dependencies.py:21]: well it SHOULD only be run once to be fair

**Comment by roberts01** [comment]
Now good?

**Comment by YuvrajRaghuvanshiS** [comment]
Just let me check both and I'm merging them.

**Comment by roberts01** [comment]
> Just let me check both and I'm merging them.

of course, thanks!

**Comment by YuvrajRaghuvanshiS** [comment]
Yes and next time please don't make more than one pull for similar issues, they get very hard to track.

**Comment by roberts01** [comment]
ok

File: helpers/termux_dependencies.py [hunk 4]
```diff
@@ -4,19 +4,21 @@
...
-    CustomPrint("Done. Installing Java for termux.")
     try : 
-        os.system('rm -rf installjava') # Deleting any previous instance of installjava.
+        os.system('rm -r -f installjava') # Deleting any previous instance of installjava.
     except Exception as e : 
         pass
-    os.system('wget https://raw.githubusercontent.com/Hax4us/java/master/installjava && sh installjava')
-    CustomPrint("Done installing all dependencies. Now running proot.")
+    os.system('wget https://raw.githubusercontent.com/MasterDevX/java/master/installjava && sh installjava')
+    os.system('proot login')
```
Comment (YuvrajRaghuvanshiS) [inline helpers/termux_dependencies.py:19]: Proot login should be run every time script is run and not just when getting dependencies.

Comment (YuvrajRaghuvanshiS) [reply helpers/termux_dependencies.py:19]: You may add that in documentations later on. Merging anyway.

**Review by YuvrajRaghuvanshiS — APPROVED** [review]

**Pull request merged**